api/client.php

```diff
@@ -1,4 +1,5 @@
 <?php
+    require_once '../config.php';
```

Comment:
I don't like RELATIVE path especially for those are not in the same directory.
The more we use `../`, the code base will be harder to maintain.
I prefer to use `require  $_SERVER["DOCUMENT_ROOT"].'/config.php'`
If you have more elegent suggestions, feel free to challenge me. :-)